firebase-test/firebasetest.js

```diff
@@ -33,3 +48,45 @@ dbUsername.on("value", function(dataSnapshot) {
 	usernameBox.textContent = dataSnapshot.val();
```

Comment:
Sorry I must have missed this in your last pull request -- you actually should have **two** Firebase event listeners -- one for the greeting, and one for the username. So you should remove line 48, then copy lines 40 to 50, paste a duplicate, and change that copy to listen for changes to your *other* Firebase reference (`dbUsername`) and modify your *other* paragraph (`usernameBox`) accordingly.